### PR TITLE
Fix filesystem doc

### PR DESCRIPTION
**Pull request by MocioF**

### CODE REVIEW

**Comment by MocioF** [comment]
have read the CLA Document and I hereby sign the CLA

**Comment by MocioF** [comment]
recheck

File: docs/persisters/filesystem.md [hunk 1]
```diff
@@ -8,14 +8,33 @@ Filesystems are local or remote storage drives that are organized by files and f
...
+$encoding = $serializer->serialize($persistable);
+$persister->save($encoding);
+
+```
+
+## Example
+```php
+use Rubix\ML\Persisters\Filesystem;
+use Rubix\ML\Serializers\RBX;
+use Rubix\ML\Classifiers\KNearestNeighbors;
```
Comment (andrewdalpino) [inline docs/persisters/filesystem.md]: Looks like these next two imports are unused in the block of code below. If so, let's remove them.

File: docs/persisters/filesystem.md [hunk 2]
```diff
@@ -8,14 +8,33 @@ Filesystems are local or remote storage drives that are organized by files and f
...
+use Rubix\ML\Kernels\Distance\Manhattan;
 
-$persister = new Filesystem('/path/to/example.model', true, new RBX());
+$persistable = new KNearestNeighbors(3, false, new Manhattan());
+$persister = new Filesystem('/path/to/example.rbx', true);
+$serializer = new RBX(6);
+$encoding = $serializer->serialize($persistable);
+$persister->save($encoding);
+
```
Comment (andrewdalpino) [inline docs/persisters/filesystem.md]: Looks like an extra line break here, if so let's remove it.

**Review by andrewdalpino — CHANGES_REQUESTED** [review]
Good start!

**Review by andrewdalpino — APPROVED** [review]

**Pull request merged**